ERCS/erc-7730.md

```diff
@@ -0,0 +1,442 @@
+* A list of `required` fields, as an array of *pathes* referring to specific fields. Wallets SHOULD display at least all the fields in the `required` key.
+* Specific device grouping and formatting information, in the `screens` key. The structure of the `screens` sub-keys are wallet maker specific and refered to in the [Wallets](#wallets) section
+* A list of `fields`. Each `fields` key name is a [path](#path-references) in the structured data, and the value is either
+  * A reference to a common definition in the `definitions` section, by using the `$ref` sub-key with a path to the internal definition. A reference can override a specification `params` by including its own `params` sub-key, whose values will take precedence over the common definition
+  * A *field format definition*, described just after
+  * A recursive sub-structure format definition, by including another list of pathes under a `fields` sub-key. This allows structuring the *field format definitions* themselves, but is NOT RECOMMENDED to maximize compatibility with resource limited wallets. All relative pathes in nested structures (fields but also in format parameters) are relative to the substructure referenced by the path in the key.  
+
+A *all relative to the substructure referenced by the path in the key* The `label` is a displayable string that should be shown before displaying this field
+* The `format` key indicates how the value of the field should be formatted before being displayed. The list of supported formats are in the [Reference](#field-formats) section
+* Each field format might have parameters in a `params` sub-key. Available parameters are descried in the [Reference](#field-formats) section
+* Each field definition can have an optional internal `$id` used to identify easily which field is described 
+
+*Examples*
+
+
+## Reference
+
+### Container structure values
+
+This section describes all container structure supported by this specification and possible references path to relevant values.
+
+#### EVM Transaction container
+
+| Value reference | Value Description | Examples |
+| --- | --- | --- |
+| @.value        | The native currency value of the transaction containing the structured data | |
+
+### Field formats
+
+#### Integer formats
+
+Formats useable for uint/int solidity types 
+
+| Format name | Parameters | Format Description | Examples |
+| ----------- | ---------- | ------------------ | -------- |
+| raw         | n/a        | Raw int value      | Value 1000 displayed as `1000` |
+| amount      | n/a        | Display as an amount in native currency, using best ticker / magnitude match.    | Value 0x2c1c986f1c48000is displayed as `0.19866144 Eth` |
+| tokenAmount | tokenPath  | Convert value using magnitude of token, and append ticker name.    | Value 1000000, for ticker DAI, magnitude 6 displayed as `1 DAI` |
```

Comment:
Is magnitude the same as decimals? I think the latter is used a bit more commonly.

Reply:
Indeed, same, will change the name